host/xtest/ffa_spmc_1000.c

```diff
@@ -160,3 +179,292 @@ static void xtest_ffa_spmc_test_1001(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(ffa_spmc, 1001, xtest_ffa_spmc_test_1001,
 		 "Test FF-A communication");
+
+static void check_alive(ADBG_Case_t *c, uint16_t endpoint)
+{
+	struct ffa_ioctl_msg_args args = {0};
+
+	args.dst_id = endpoint;
+	ADBG_EXPECT_TRUE(c, start_sp_test(endpoint, EP_SP_NOP, &args));
+	ADBG_EXPECT_TRUE(c, check_ok(&args));
+}
+
+static bool share_mem(uint16_t endpoint, uint64_t *handle)
+{
+	bool status = false;
+	struct ffa_ioctl_shm_desc shm_desc = {.dst_id = endpoint,
```

Comment:
Add blank line here

Reply:
@jforissier there is already a blank line after the struct declaration. Am I missing something?

Reply:
My mistake, sorry